easybuild/tools/multidiff.py

```diff
@@ -224,7 +227,7 @@ def limit(text, length):
 
         diff = False
         for i in range(len(self.base_lines)):
-            lines = filter(None, self.get_line(i))
+            lines = [line for line in self.get_line(i) if line]
```

Comment:
is this exactly the same? Shouldn't you use `if line is not None`?

Reply:
No, with `None` as first element to `function` this is exactly equivalent, it's even mentioned explicitly to be so in the documentation: https://docs.python.org/2/library/functions.html#filter and https://docs.python.org/3/library/functions.html#filter